test/e2e/specs/writing-flow.test.js

```diff
@@ -149,13 +149,11 @@ describe( 'adding blocks', () => {
 		await pressWithModifier( META_KEY, 'b' );
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 
-		// When returning to Visual mode, backspace in selected block should
-		// reset to an unmodified default block.
+		// Reset.
 		await page.keyboard.press( 'Backspace' );
 
 		// Ensure no data-mce-selected. Notably, this can occur when content
 		// is saved while typing within an inline boundary.
-		await clickBlockAppender();
```

Comment:
Why the change? I believe you, but it's hard to tell because there isn't accompanying code in the PR.

Reply:
Before #9977, it was possible to remove all blocks. The default appender was shown.

After #9977, removing all blocks causes an empty default block to be inserted, at which point the default appender is not present on the page (hence the original error).

Reply:
👍